app/assets/javascripts/search.js

```diff
@@ -13,7 +13,11 @@ function search() {
         var searchTitle = title.indexOf(searchInput) >= 0;
         var searchBody  = body.indexOf(searchInput) >= 0;
 
-        (searchTitle || searchBody) ? $(this).show() : $(this).hide();
```

Comment:
I don't know what the community opinion is - I personally love using ternaries. We could ask in #questions and see if it's one of those hot button topics like tabs or spaces.....